can/io/sqlite.py

```diff
@@ -65,10 +68,8 @@ def close(self):
         """Closes the connection to the database."""
         self.conn.close()
 
-
-# Backward compatibility
-# TODO remove in later releases?
-SqlReader = SqliteReader
+# SqliteReader is the newer name
+SqliteReader = SqlReader
```

Comment:
Won't the `SqliteReader` also be marked as deprecated with this approach?

Reply:
Yes yes, it all doesn't work like I intended. But de we really need this deprecation at all? We already broke compatibility.

Reply:
Hmm I didn't realize we'd already broken compatibility - in that case no we don't really need this deprecation warning. Lets just close this PR and remove the SqlReader/SqliteReader alias.